src/Http/Controllers/AjaxMultiselectController.php

```diff
@@ -20,7 +20,10 @@ public function getOptionsByQuery(NovaRequest $request)
 
         $builder->where($label, 'like', "%$query%");
         if ($custom_query) {
-            $builder->whereRaw($custom_query);
+            $custom_query = parse_str($custom_query, $query);
```

Comment:
@Stowez guessing this will might break something we're we use this to write custom queries?

Reply:
@JKHarley I'd recommend adding a new GET param to handle these kind of key => value queries to keep backward compatibility

Reply:
> @Stowez guessing this will might break something we're we use this to write custom queries?

@rossjcooper Got the wrong end of the stick here? 🤔

Reply:
I'm not sure mate which is why I tagged @Stowez to double check seen as its originally using a whereRaw